samples/bluetooth/broadcast_audio_source/Kconfig

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Demant A/S
+# SPDX-License-Identifier: Apache-2.0
+
+
+mainmenu "Bluetooth: Broadcast Audio Source"
+
+choice BT_BAP_LC3_PRESET
```

```suggestion
choice BAP_LC3_PRESET
```

We should avoid using the `BT_` prefix in sample Kconfigs

---

Okay, I'll keep it in mind! But could you help me understand why that is the case? 
I imagine that's so that the Kconfig can be used instead in non-sample projects... however if a Kconfig is defined in a project's Kconfig file, isn't it the case that it is only in project scope, thus not "interfering" with other projects' Kconfigs?

---

The Bluetooth subsystem basically reserves the `BT_`/`bt_` prefix, so that when we add/change Kconfig options in the subsystem, we don't have to worry about applications/samples already using the names. 

So if you add `BT_BAP_LC3_PRESET` here, then that effectively mean that we cannot use that name in the subsystem. While it is unlikely we would have this specific name, we should generally avoid using subsystem prefixes in the samples

---

Fantastic. That makes a lot of sense, thanks 👍🏻